libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.scss

```diff
@@ -1,157 +1,306 @@
-  background-color: $sky-background-color-neutral-light;
-  padding: 1px;
-  border-radius: $indicator-size + 2;
-  flex: none;
-  line-height: $sky-line-height-base;
-  width: $indicator-size * 2 + 2;
-  display: inline-block;
-}
 
-.sky-toggle-switch-button {
-  border: none;
-  background: none;
-  margin: 0;
-  padding: 0;
-  outline: none;
-
-  &:focus-visible .sky-toggle-switch-switch {
-    outline: none;
-    border: solid 2px $sky-theme-modern-background-color-primary-dark;
+  .sky-toggle-switch-button {
+    border: none;
+    background: none;
+    margin: 0;
     padding: 0;
-  }
-
-  &:hover:not(.sky-toggle-switch-disabled) {
-    cursor: pointer;
+    outline: none;
 
-    &:not(:focus-visible) .sky-toggle-switch-switch {
-      border-color: $sky-highlight-color-info;
-      border-width: 2px;
-      padding: 0;
+    .sky-toggle-switch-switch {
+      border: var(
+        --sky-override-toggle-switch-border,
+        var(--sky-border-width-input-base) solid
+          var(--sky-color-border-input-base)
+      );
+      background-color: var(
+        --sky-override-toggle-switch-background-color,
+        var(--sky-background-input-base)
+      );
+      padding: var(
+        --sky-override-toggle-switch-padding,
+        var(--sky-border-width-input-base)
+      );
+      border-radius: var(
+        --sky-override-toggle-switch-border-radius,
+        calc(
+            var(--sky-border-radius-thumb) +
+              calc(var(--sky-space-inset-thumb) * 2)
+          ) / 100%
+      );
+      flex: none;
+      line-height: $sky-line-height-base;
+      width: var(
+        --sky-override-toggle-switch-width,
+        calc(
+          calc(var(--sky-size-switch) * 2) - calc(
+              calc(var(--sky-border-width-input-base) * 2) +
+                calc(var(--sky-space-inset-thumb) * 2)
+            )
+        )
+      );
+      display: inline-block;
+      transition: var(
+        --sky-override-toggle-switch-transition,
+        border-color $sky-transition-time-short,
+        box-shadow $sky-transition-time-short
+      );
     }
-  }
 
-  &.sky-toggle-switch-checked {
-    .sky-toggle-switch-switch {
-      background-color: $sky-background-color-info-light;
-      border-width: 2px;
-      padding: 0;
+    &:focus-visible .sky-toggle-switch-switch {
+      border-width: var(
+        --sky-override-toggle-switch-border-width-focus,
+        var(--sky-border-width-input-focus)
+      );
+      border-color: var(
+        --sky-override-toggle-switch-border-color-focus,
+        var(--sky-color-border-input-focus)
+      );
+      padding: var(
+        --sky-override-toggle-switch-padding-focus,
+        calc(
+          calc(var(--sky-space-inset-thumb) * 2) - var(
+              --sky-border-width-input-focus
+            )
+        )
+      );
     }
 
-    &:not(:focus-visible, .sky-toggle-switch-disabled)
+    &:focus-visible:not(:is(:active, .sky-toggle-switch-disabled))
```

Comment:
I think you can pass a comma separated list to `:not`, so `&:focus-visible:not(:active, .sky-toggle-switch-disabled)`